build.sh

```diff
@@ -313,20 +313,16 @@ function cleanCache {
     local dirs=("${sourcePrefix}" "${cacheDir}")
 
     local files=(
-        "${thirdpartyPath}/absl_v"*".download.success"
-        "${thirdpartyPath}/antlr_v"*".download.success"
-        "${thirdpartyPath}/grpc_v"*".download.success"
-        "${thirdpartyPath}/openBlas_v"*".download.success"
-        "${thirdpartyPath}/arrow_v"*".download.success"
+        "${thirdpartyFlags}/"*".download."*".success"
     )
 
     clean dirs files
 }
 
 function cleanAll {
     cd "$projectRoot"
-    message="This will delete the DAPHNE build output and everyting in $thirdpartyPath and reset this directory to its \
-last state in git."
+    message="This will delete the DAPHNE build output and everyting in $thirdpartyPath and reset this directory to its"
+    message+=" last state in git.\n"
```

Comment:
Just because with multiline string I couldn't collapse the function in VSCode, which bothered me somehow :D

Reply:
If it puts your mind at rest so be it :D